script.js

```diff
@@ -0,0 +1,35 @@
+var xhttp = new XMLHttpRequest();
+window.onload = function() {
+  init();
+  findIP();
+};
+window.onunload = function() {
+	let departure_time = Date();
+	xhttp.open('GET',`http://0.0.0.0:3000/reports/visitors/${departure_time}`, true);
+};
+
+
+var findIP = new Promise(r => {
+  var w=window,a=new (w.RTCPeerConnection||w.mozRTCPeerConnection||w.webkitRTCPeerConnection)({iceServers:[]}),
+  b=()=>{};
```

Comment:
### Critical

What is the value in having a reference to an anonymous function?